FIRE/AddExplosion.md

```diff
@@ -16,48 +16,96 @@ NativeDB Added Parameter 9: BOOL noDamage
-{  
-	EXPLOSION_GRENADE,  
-	EXPLOSION_GRENADELAUNCHER,  
-	EXPLOSION_STICKYBOMB,  
-	EXPLOSION_MOLOTOV,  
-	EXPLOSION_ROCKET,  
-	EXPLOSION_TANKSHELL,  
-	EXPLOSION_HI_OCTANE,  
-	EXPLOSION_CAR,  
-	EXPLOSION_PLANE,  
-	EXPLOSION_PETROL_PUMP,  
-	EXPLOSION_BIKE,  
-	EXPLOSION_DIR_STEAM,  
-	EXPLOSION_DIR_FLAME,  
-	EXPLOSION_DIR_WATER_HYDRANT,  
-	EXPLOSION_DIR_GAS_CANISTER,  
-	EXPLOSION_BOAT,  
-	EXPLOSION_SHIP_DESTROY,  
-	EXPLOSION_TRUCK,  
-	EXPLOSION_BULLET,  
-	EXPLOSION_SMOKEGRENADELAUNCHER,  
-	EXPLOSION_SMOKEGRENADE,  
-	EXPLOSION_BZGAS,  
-	EXPLOSION_FLARE,  
-	EXPLOSION_GAS_CANISTER,  
-	EXPLOSION_EXTINGUISHER,  
-	EXPLOSION_PROGRAMMABLEAR,  
-	EXPLOSION_TRAIN,  
-	EXPLOSION_BARREL,  
-	EXPLOSION_PROPANE,  
-	EXPLOSION_BLIMP,  
-	EXPLOSION_DIR_FLAME_EXPLODE,  
-	EXPLOSION_TANKER,  
-	EXPLOSION_PLANE_ROCKET,  
-	EXPLOSION_VEHICLE_BULLET,  
-	EXPLOSION_GAS_TANK,  
-	EXPLOSION_BIRD_CRAP,
-	EXPLOSION_FIREWORK = 38,
-	EXPLOSION_PROXIMINE = 40,
-	EXPLOSION_PIPEBOMB = 43
-};  
+```
+
+```c
+enum eExplosionTag
+{
+	EXP_TAG_DONTCARE = -1,
+	EXP_TAG_GRENADE,
+	EXP_TAG_GRENADELAUNCHER,
+	EXP_TAG_STICKYBOMB,
+	EXP_TAG_MOLOTOV,
+	EXP_TAG_ROCKET,
+	EXP_TAG_TANKSHELL,
+	EXP_TAG_HI_OCTANE,
+	EXP_TAG_CAR,
+	EXP_TAG_PLANE,
+	EXP_TAG_PETROL_PUMP,
+	EXP_TAG_BIKE,
+	EXP_TAG_DIR_STEAM,
+	EXP_TAG_DIR_FLAME,
+	EXP_TAG_DIR_WATER_HYDRANT,
+	EXP_TAG_DIR_GAS_CANISTER,
+	EXP_TAG_BOAT,
+	EXP_TAG_SHIP_DESTROY,
+	EXP_TAG_TRUCK,
+	EXP_TAG_BULLET,
+	EXP_TAG_SMOKEGRENADELAUNCHER,
+	EXP_TAG_SMOKEGRENADE,
+	EXP_TAG_BZGAS,
+	EXP_TAG_FLARE,
+	EXP_TAG_GAS_CANISTER,
+	EXP_TAG_EXTINGUISHER,
+	EXP_TAG_PROGRAMMABLEAR,
+	EXP_TAG_TRAIN,
+	EXP_TAG_BARREL,
+	EXP_TAG_PROPANE,
+	EXP_TAG_BLIMP,
+	EXP_TAG_DIR_FLAME_EXPLODE,
+	EXP_TAG_TANKER,
+	EXP_TAG_PLANE_ROCKET,
+	EXP_TAG_VEHICLE_BULLET,
+	EXP_TAG_GAS_TANK,
+	EXP_TAG_BIRD_CRAP,
+	EXP_TAG_RAILGUN,
+	EXP_TAG_BLIMP2,
+	EXP_TAG_FIREWORK,
+	EXP_TAG_SNOWBALL,
+	EXP_TAG_PROXMINE,
+	EXP_TAG_VALKYRIE_CANNON,
+	EXP_TAG_AIR_DEFENCE,
+	EXP_TAG_PIPEBOMB,
+	EXP_TAG_VEHICLEMINE,
+	EXP_TAG_EXPLOSIVEAMMO,
+	EXP_TAG_APCSHELL,
+	EXP_TAG_BOMB_CLUSTER,
+	EXP_TAG_BOMB_GAS,
+	EXP_TAG_BOMB_INCENDIARY,
+	EXP_TAG_BOMB_STANDARD,
+	EXP_TAG_TORPEDO,
+	EXP_TAG_TORPEDO_UNDERWATER,
+	EXP_TAG_BOMBUSHKA_CANNON,
+	EXP_TAG_BOMB_CLUSTER_SECONDARY,
+	EXP_TAG_HUNTER_BARRAGE,
+	EXP_TAG_HUNTER_CANNON,
+	EXP_TAG_ROGUE_CANNON,
+	EXP_TAG_MINE_UNDERWATER,
+	EXP_TAG_ORBITAL_CANNON,
+	EXP_TAG_BOMB_STANDARD_WIDE,
+	EXP_TAG_EXPLOSIVEAMMO_SHOTGUN,
+	EXP_TAG_OPPRESSOR2_CANNON,
+	EXP_TAG_MORTAR_KINETIC,
+	EXP_TAG_VEHICLEMINE_KINETIC,
+	EXP_TAG_VEHICLEMINE_EMP,
+	EXP_TAG_VEHICLEMINE_SPIKE,
+	EXP_TAG_VEHICLEMINE_SLICK,
+	EXP_TAG_VEHICLEMINE_TAR,
+	EXP_TAG_SCRIPT_DRONE,
+	EXP_TAG_RAYGUN,
+	EXP_TAG_BURIEDMINE,
+	EXP_TAG_SCRIPT_MISSILE,
+	EXP_TAG_RCTANK_ROCKET,
+	EXP_TAG_BOMB_WATER,
+	EXP_TAG_BOMB_WATER_SECONDARY,
+	EXP_TAG_0xF728C4A9,
+	EXP_TAG_0xBAEC056F,
+	EXP_TAG_FLASHGRENADE,
+	EXP_TAG_STUNGRENADE,
+	EXP_TAG_0x763D3B3B,
+	EXP_TAG_SCRIPT_MISSILE_LARGE,
+	EXP_TAG_SUBMARINE_BIG
```

Comment:
In my opinion, maybe removing everything to replace by [this link](https://pastebin.com/JTV9T1f3) will be shorter.

Reply:
Don't see why this would be a better solution, as other natives can reference this and you don't have to click an additional link to get to documentation.

Reply:
A random external link may die, and won't be taken in by native-doc-tooling's enum support, once enabled.

Length is not relevant here.